### ISSUE TITLE
Delete multiple issue

### ISSUE THREAD

**Issue by miyuki-ts** [Piotrekol/CollectionManager#83]
If i try to select multiple at once it only delete the first one. I just reinstall and am on the newest version. it still happens. I used to work, idk why it does not work anymore, and i hate hit delete 500 times or more instead of just delete them all at once.

![image](https://github.com/user-attachments/assets/956db254-173e-4283-8ae8-9202efdc1aef)

**Comment by Piotrekol** [comment]
Can not reproduce on latest release - You'll have to provide exact reproduction steps.

**Comment by Piotrekol** [comment]
No. Provide step by step instructions to reproduce this issue. 
Also: What system are you running? Does this happen only with specific collection or specific collection format?
